src/webapp/setup.py

```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "0.3.0"
+VERSION = "0.3.1"
```

Comment:
pls also update history.rst, tricky part there's no change in the new version. maybe you don't need to increment to 0.3.1, we should have way to retrigger release pipeline. @qwordy  @haroldrandom is it possible to trigger a new release PR for existing code without increment version?

Reply:
Yeah it will be great if we can trigger a new run of the pipeline with the latest commits included in it. The previous run failed because of errors that have been fixed now: https://github.com/Azure/azure-cli-extensions/pull/2781

Reply:
I have updated the PR to include HISTORY.rst. If you can retry releasing 0.3.0 that would be great. Let me know how it goes and I'll add the HISTORY.rst separately in that case as soon as 0.3.0 is published.